source/Ordinals/MultiplicationProperties.lagda

```diff
@@ -0,0 +1,489 @@
+simulation-product-decomposition {𝓤} {𝓥} α β γ (a₀ , a₀-least)
+                                 (f , sim@(init-seg , order-pres)) a b = I
+ where
+  f' : ⟨ α ×ₒ β ⟩ → ⟨ α ×ₒ γ ⟩
+  f' (a , b) = (a , pr₂ (f (a₀ , b)))
+
+  P : ⟨ α ×ₒ β ⟩ → 𝓤 ⊔ 𝓥 ̇
+  P (a , b) = (f (a , b)) ＝ f' (a , b)
+
+  I : P (a , b)
+  I = Transfinite-induction (α ×ₒ β) P II (a , b)
+   where
+    II : (x : ⟨ α ×ₒ β ⟩)
+       → ((y : ⟨ α ×ₒ β ⟩) → y ≺⟨ α ×ₒ β ⟩ x → P y)
+       → P x
+    II (a , b) IH = Extensionality (α ×ₒ γ) (f (a , b)) (f' (a , b)) III IV
+     where
+      III : (u : ⟨ α ×ₒ γ ⟩) → u ≺⟨ α ×ₒ γ ⟩ f (a , b) → u ≺⟨ α ×ₒ γ ⟩ f' (a , b)
+      III (a' , c') p = transport (λ - → - ≺⟨ α ×ₒ γ ⟩ f' (a , b)) III₂ (III₃ p')
+       where
+        III₁ : Σ (a'' , b') ꞉ ⟨ α ×ₒ β ⟩ , (a'' , b') ≺⟨ α ×ₒ β ⟩ (a , b)
+                                         × (f (a'' , b') ＝ a' , c')
+        III₁ = init-seg (a , b) (a' , c') p
+        a'' = pr₁ (pr₁ III₁)
+        b' = pr₂ (pr₁ III₁)
+        p' = pr₁ (pr₂ III₁)
+        eq : f (a'' , b') ＝ (a' , c')
+        eq = pr₂ (pr₂ III₁)
+
+        III₂ : f' (a'' , b') ＝ (a' , c')
+        III₂ = IH (a'' , b') p' ⁻¹ ∙ eq
+
+        III₃ : (a'' , b') ≺⟨ α ×ₒ β ⟩ (a , b)
+             → f' (a'' , b') ≺⟨ α ×ₒ γ ⟩ f' (a , b)
+        III₃ (inl q) = h (order-pres (a₀' , b') (a₀ , b) (inl q))
+         where
+          a₀' : ⟨ α ⟩
+          a₀' = pr₁ (f (a₀ , b))
+
+          ih : (f (a₀' , b')) ＝ f' (a₀' , b')
+          ih = IH (a₀' , b') (inl q)
+
+          h : f  (a₀' , b') ≺⟨ α ×ₒ γ ⟩ f  (a₀ , b)
+            → f' (a'' , b') ≺⟨ α ×ₒ γ ⟩ f' (a , b)
+          h (inl r) = inl (transport (λ - → - ≺⟨ γ ⟩ pr₂ (f (a₀ , b)))
+                                     (ap pr₂ ih) r)
+          h (inr (_ , r)) = 𝟘-elim (irrefl α a₀' (transport (λ - → - ≺⟨ α ⟩ a₀')
+                                                            (ap pr₁ ih) r))
+        III₃ (inr (e , q)) = inr (ap (λ - → pr₂ (f (a₀ , -))) e , q)
+
+      IV : (u : ⟨ α ×ₒ γ ⟩) → u ≺⟨ α ×ₒ γ ⟩ f' (a , b) → u ≺⟨ α ×ₒ γ ⟩ f  (a , b)
+      IV (a' , c') (inl p) = l₂ (a' , c') (inl p)
+       where
+        l₁ : a₀ ≼⟨ α ⟩ a
+        l₁ x p = 𝟘-elim (transport ⟨_⟩ (a₀-least ⁻¹) (x , p))
+        l₂ : f (a₀ , b) ≼⟨ α ×ₒ γ ⟩ f (a , b)
+        l₂ = simulations-are-monotone _ _
+              f sim (a₀ , b) (a , b) (×ₒ-≼-left α β l₁)
+      IV (a' , c') (inr (r , q)) =
+       transport (λ - → - ≺⟨ α ×ₒ γ ⟩ f  (a , b)) eq
+                 (order-pres (a' , b) (a , b) (inr (refl , q)))
+        where
+         eq = f  (a' , b)             ＝⟨ IH (a' , b) (inr (refl , q)) ⟩
+              f' (a' , b)             ＝⟨ refl ⟩
+              (a' , pr₂ (f (a₀ , b))) ＝⟨ ap (a' ,_) (r ⁻¹) ⟩
+              (a' , c')               ∎
+
+\end{code}
+
+The following result states that multiplication for ordinals can be
+cancelled on the left. Interestingly, Andrew Swan [Swa18] proved that
+the corresponding result for mere sets is not provable constructively
```

Comment:
I am not requesting a change. But "mere sets" can be confusing given that some people use "merely" to mean propositional truncation. You could just say "sets".